docs/pages-for-subheaders/azure-marketplace-payg-integration.md

```diff
@@ -0,0 +1,227 @@
+
+#### I already have an existing AKS cluster in place and want to add Rancher Prime to it and be billed through the Azure Marketplace; is this possible?
+
+Yes, simply deploy the Rancher Prime to the cluster with the Azure Marketplace listing.
+
+#### I already deployed Rancher to an existing AKS cluster; can I just install the marketplace version to enable Azure Marketplace billing?
+
+No. You need to deploy Rancher Prime with the Azure Marketplace listing and migrate the existing Rancher configuration to this new deployment. Be sure that you back up your existing Rancher configuration.
+
+#### How do I get support?
+
+It is very simple to open a support case with SUSE for Rancher Prime. Create a ‘supportconfig’ via the Rancher UI (click Get Support under the hamburger menu and follow instructions), then upload the ‘supportconfig’ output to the SUSE Customer Center. If the billing mechanism is active, a support case will be opened. See Supportconfig bundle in the Rancher documentation for more details.
+
+:::note
+
+For deployments where Rancher Prime is managing multiple downstream clusters, be sure to export the ‘supportconfig’ bundle from the primary cluster only.
+
+:::
+
+#### What are the resource requirements for installing Rancher on AKS?
+
+Check the official documentation for best practices. Is there any difference between Rancher Prime from the Azure
+
+#### Marketplace and the versions I can run in my own data center?
+
+Rancher Prime available in the Azure Marketplace is the same product, with the same functionality that you would install manually in the cloud or on-premises.The only difference between deploying manually and deploying via the marketplace listing is the billing route.
+
+#### Does the primary cluster (responsible for billing) need to run 24/7?
+
+To ensure continuity of support, it is recommended that the primary Rancher Prime cluster always remain active.
+
+#### What if the primary cluster responsible for billing is unable to connect to the Azure billing framework?
+
+There may be multiple reasons why the primary cluster is unable to connect to the Azure billing framework, but it is the customer’s responsibility to ensure that the primary cluster is active and connected. While the cluster is not connected to the billing framework, it is not possible to raise a support request.
+
+#### My primary cluster has been offline; what will happen with billing when reconnected?
+
+If the Racher Prime cluster is offline or disconnected from the Azure billing framework for a period of time, when it reconnects, the stored usage data will be uploaded to Azure and will appear on your next Azure bill. Depending on the month when the primary cluster gets reconnected you may have several months of usage on your next billing cycle.
+
+#### How do I get fixes and updates for Rancher?
+
+To update the Rancher product to a current version before the marketplace listing is updated, see Upgrades in the official documentation.
```

Comment:
customers cannot do this, they can get updates as the official listings are updated, for Azure they need to use the Azure specific update mechanism documented under "Upgrading a Rancher Prime PAYG Cluster
" at https://suse-enceladus.github.io/marketplace-docs/rancher-prime/azure/   (yes, we need to add anchors to the headings there...)